axelrod/tests/unit/test_utils.py

```diff
@@ -41,3 +41,18 @@ def test_basic_configuration_console(self):
             axelrod.utils.setup_logging(logging_destination='console',
                                         verbosity=level)
             compare(logger.level, levels[level])
+
+
+class TestBuildExclusionsDict(unittest.TestCase):
+    def test_build_exclusions_dict(self):
+        """This is really a regression test to make sure the names of the
+        tournaments stay consistent"""
+        bools = [True, False]
+        for (eb, eo, ec, eco) in itertools.product(*[bools for _ in range(4)]):
```

Comment:
This is a lot to unpack -- is there another way to express the argument of the product? I think there is a repeat keyword argument that may do the same thing.